test/performance/benchmarks/dataplane-probe/continuous/dataplane-probe-setup.yaml

```diff
@@ -1,228 +0,0 @@
-# Copyright 2022 The Knative Authors
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     https://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-apiVersion: serving.knative.dev/v1
-kind: Service
-metadata:
-  name: activator
-  namespace: default
-spec:
-  template:
-    metadata:
-      annotations:
-        autoscaling.knative.dev/minScale: "5"
-        autoscaling.knative.dev/maxScale: "5"
-        # Always hook the activator in.
-        autoscaling.knative.dev/targetBurstCapacity: "-1"
-    spec:
-      containers:
-      - image: ko://knative.dev/serving/test/test_images/autoscale
-      containerConcurrency: 0 # Explicitly set the default, since it might be overridden in CM.
----
-apiVersion: serving.knative.dev/v1
-kind: Service
-metadata:
-  name: activator-with-cc
```

Comment:
Any reason why we're dropping the different benchmarks for different activator/queue-proxy configurations?

Reply:
They were never used/called by code or at least I could not find a job that would actually test against them. Also I could not figure out what the original intention could have been.

The existing jobs just called:
- [depoyment](https://github.com/knative/serving/pull/14289/files/6d06e81f51f9da8599040c01b23d19c642fc1bee#diff-f39f781bb0e5766c492d389e5aa16d349cdeac197124509a2a84e16cdda8ec23L50)
- [istio](https://github.com/knative/serving/pull/14289/files/6d06e81f51f9da8599040c01b23d19c642fc1bee#diff-f39f781bb0e5766c492d389e5aa16d349cdeac197124509a2a84e16cdda8ec23L92) I dropped that, as its a bit weird that our generic tests test a specific ingress layer. As these tests should also run against kourier + contour
- [queue](https://github.com/knative/serving/pull/14289/files/6d06e81f51f9da8599040c01b23d19c642fc1bee#diff-f39f781bb0e5766c492d389e5aa16d349cdeac197124509a2a84e16cdda8ec23L134)
- [activator](https://github.com/knative/serving/pull/14289/files/6d06e81f51f9da8599040c01b23d19c642fc1bee#diff-f39f781bb0e5766c492d389e5aa16d349cdeac197124509a2a84e16cdda8ec23L176)

Reply:
> They were never used/called by code or at least I could not find a job that would actually test against them.

Rollout probe https://github.com/knative/serving/blob/4cb442c33871b1ddb514ba3b9940de6ceaf985b1/test/performance/benchmarks/rollout-probe/continuous/rollout-probe-activator-direct.yaml#L49

The rollout probe would ensure we're not dropping requests when updating Knative Services

Reply:
We still have that one: https://github.com/knative/serving/pull/14289/files#diff-f670c78672a9d94e0786f79912856a6dd8576dd3bffe3bdfa33036ada40fc2a9R49

I meant we haven't have called it for the `dataplane-probe`. I kept them for the `rollout-probe`.